packages/react-native-gesture-handler/apple/RNGestureHandlerEvents.mm

```diff
@@ -176,6 +176,7 @@ - (instancetype)initWithReactTag:(NSNumber *)reactTag
                            state:(RNGestureHandlerState)state
                        extraData:(RNGestureHandlerEventExtraData *)extraData
                    forActionType:(NSInteger)actionType
```

Comment:
Is action type needed now in the events?

Reply:
Seems like it is not. Removed in [9cc3073](https://github.com/software-mansion/react-native-gesture-handler/pull/3646/commits/9cc3073d4150eb434026748ee729e45a973a9d7f)